contrib/buildpkgs.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+[ "x${WORKSPACE}" == "x" ] && { echo "Please set WORKSPACE"; exit 1; }
+
+C_DIR=${PWD}
+
+DEST_DIR="${C_DIR}/PKGS"
+
+[ ! -z "${GHNL}" ] && . ${GHNL}
+
+if [ -x /usr/bin/dpkg-buildpackage ] ; then
+	PKGM="deb"
+	PKG_DIR=${C_DIR}/deb
+else
+	PKGM="rpm"
+	PKG_DIR=${C_DIR}/rpm
+	SRC_PKG_DIR="${PKG_DIR}/SRC"
+	PKG_SPEC=${WORKSPACE}/build/libvma.spec
```

Comment:
In the _make_dist()_  _${WORKSPACE}/configure_ is used and then _make dist_. And if _make dist_ returns an error or _libvma.X.X.X.tar.gz_ doesn't exist then the script finished with _err "file not found"_